hack/dockerfiles/lint.Dockerfile

```diff
@@ -1,8 +1,11 @@
 # syntax=docker/dockerfile:1
 
-FROM golang:1.18-alpine
+ARG GO_VERSION=1.19
+ARG ALPINE_VERSION=3.17
+
+FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION}
```

Comment:
Is there a point in configuring alpine version here? The alpine part of these stages is not really used and probably only certain configurations are available anyway.

Reply:
Actually no. Removed.